tests/fv3_conf/ccpp_regional_run.IN

```diff
@@ -2,9 +2,18 @@ rm -fr INPUT RESTART
 mkdir INPUT RESTART
 rsync -av @[RTPWD]/FV3_input_data_regional/ ./
 if [ $WARM_START = .T. ]; then
-  cp ../fv3_ccpp_regional_coldstart${RT_SUFFIX}/RESTART/* INPUT/
-  cp ../fv3_ccpp_regional_coldstart${RT_SUFFIX}/INPUT/gfs_bndy.tile7.*.nc INPUT/
-  cp ../fv3_ccpp_regional_coldstart${RT_SUFFIX}/INPUT/oro_data.tile7.halo4.nc INPUT/
+  if [ $CCPP_SUITE = FV3_RRFS_v1alpha ]; then
+    cp ../fv3_ccpp_regional_coldstart${RT_SUFFIX}/RESTART/* INPUT/
+    cp ../fv3_ccpp_regional_coldstart${RT_SUFFIX}/INPUT/gfs_bndy.tile7.*.nc INPUT/
+    cp ../fv3_ccpp_regional_coldstart${RT_SUFFIX}/INPUT/oro_data.tile7.halo4.nc INPUT/
+  elif [ $CCPP_SUITE = FV3_GFS_v15p2 ]; then
+    cp ../fv3_ccpp_regional_v15p2_coldstart${RT_SUFFIX}/RESTART/* INPUT/
```

Comment:
If I understand correctly, the GFS_v15p2 warm start test runs from 12h->24h, the GFS_v15p2 control test should have restart_interval=12 in order to get the restart files for the warm start test and then those restart files should be renamed to remove the date in the restart file names. But I did not see any related code here. Can you please check?

Reply:
No, because there is a coldstart run that runs from 0 to 12h (old style before the restart_interval was introduced).

Reply:
Please see line 90 below in tests/tests/fv3_ccpp_gfs_v15p2, the control test has FHMAX=24, so the control is 0->24

Reply:
Thanks, @junwang-noaa. I know this is less efficient, because it requires running one more test. But since this should be the last PR before the code is tagged for the SRW App release, it doesn't really matter.

Reply:
> Please see line 90 below in tests/tests/fv3_ccpp_gfs_v15p2, the control test has FHMAX=24, so the control is 0->24

Yes, but control is *not* used for the restart files. Please see the lines you just highlighted. It says `fv3_ccpp_regional_coldstart${RT_SUFFIX}` and `fv3_ccpp_regional_v15p2_coldstart${RT_SUFFIX}`, and not `fv3_ccpp_regional${RT_SUFFIX}` and `fv3_ccpp_regional_v15p2${RT_SUFFIX}`.

Reply:
OK, thanks.